examples/dlint/control-flow.js

```diff
@@ -0,0 +1,18 @@
+class ControlFlow {
+  static isReachable(stmt) {
+    const { isReachable } = Deno.core.jsonOpSync("query_control_flow_by_span", {
```

Comment:
Nitpick: can you add `op_` prefix to names of ops? I find it a lot easier to search for op call sites

Reply:
Oh I forgot that. I'll do it tomorrow (because it's late at night here :)

Reply:
Done b6a85be